ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -569,6 +568,11 @@ def start_active_span(
             :class:`ScopeManagerShim`.
         """
 
+        current_span = get_current_span()
+
+        if child_of is None and current_span is not None:
```

Comment:
current_span will not be None as of #891

Reply:
Fixed